.circleci/config.yml

```diff
@@ -214,6 +214,18 @@ step-show-sccache-stats: &step-show-sccache-stats
     name: Check sccache stats after build
     command: $SCCACHE_PATH -s
 
+step-mksnapshot-build: &step-mksnapshot-build
+  run:
+    name: mksnapshot build
+    command: |
+      cd src
+      ninja -C out/Default electron:electron_mksnapshot_zip
+
+step-mksnapshot-store: &step-mksnapshot-store
+  store_artifacts:
+    path: src/out/Default/mksnapshot.zip
+    destination: mksnapshot
```

Comment:
An extension maybe?

Reply:
@alexeykuzmin destination is a directory:
>destination	N	String	Prefix added to the artifact paths in the artifacts API (default: the directory of the file specified in path)

Reply:
Hmm... CircleCI docs aren't the most helpful here.  I think if you put more than one file in a destination it treats it like a directory... but if there is only one file (as in this case), it ends up being a file link eg see: https://circleci.com/gh/electron/electron/65171#artifacts/containers/0, so we should change it to reflect that.

Reply:
> it ends up being a file link eg see: https://circleci.com/gh/electron/electron/65171#artifacts/containers/0

That's exactly why I asked. Thanks!